### PR TITLE
BUGFIX: ONE-3964 Mention the need to import pivotTable styles

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/109/ : SUCCESS in 1m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/1/ : SUCCESS in 6s

File: docs/ht_use_react_components_in_angular_2.x.md [hunk 1]
```diff
@@ -199,6 +199,8 @@ If you want to render some charts, do the following:
     
     ```
 
+**NOTE:** If you are using `PivotTable` component, import also `pivotTable.css` file to your global styles.
```
Comment (evgeniabogranskaya) [inline docs/ht_use_react_components_in_angular_2.x.md]: **NOTE:** If you are using the `PivotTable` component, import the `pivotTable.css` file into your global styles.

File: website/versioned_docs/version-5.0.0/ht_use_react_components_in_angular_2.x.md [hunk 2]
```diff
@@ -200,6 +200,8 @@ If you want to render some charts, do the following:
     
     ```
 
+**NOTE:** If you are using `PivotTable` component, import also `pivotTable.css` file to your global styles.
```
Comment (evgeniabogranskaya) [inline website/versioned_docs/version-5.0.0/ht_use_react_components_in_angular_2.x.md]: **NOTE:** If you are using the `PivotTable` component, import the `pivotTable.css` file into your global styles.

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/110/ : SUCCESS in 1m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/2/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/111/ : SUCCESS in 1m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/33/ : SUCCESS in 21s

**Pull request merged**